application/src/main/java/org/togetherjava/tjbot/commands/moderation/AuditCommand.java

```diff
@@ -53,27 +60,25 @@ public AuditCommand(@NotNull ModerationActionsStore actionsStore, @NotNull Confi
         this.actionsStore = Objects.requireNonNull(actionsStore);
     }
 
-    private static @NotNull MessageEmbed createSummaryMessage(@NotNull User user,
+    private static @NotNull EmbedBuilder createSummaryEmbed(@NotNull User user,
             @NotNull Collection<ActionRecord> actions) {
         return new EmbedBuilder().setTitle("Audit log of **%s**".formatted(user.getAsTag()))
             .setAuthor(user.getName(), null, user.getAvatarUrl())
             .setDescription(createSummaryMessageDescription(actions))
-            .setColor(ModerationUtils.AMBIENT_COLOR)
-            .build();
+            .setColor(ModerationUtils.AMBIENT_COLOR);
     }
 
     private static @NotNull String createSummaryMessageDescription(
             @NotNull Collection<ActionRecord> actions) {
         int actionAmount = actions.size();
-        if (actionAmount == 0) {
-            return "There are **no actions** against the user.";
-        }
 
-        String shortSummary = "There are **%d actions** against the user.".formatted(actionAmount);
+        String shortSummary = "There are **%s actions** against the user."
+            .formatted(actionAmount == 0 ? "no" : actionAmount);
```

Comment:
Why does this not return anymore? Now, it has a trailing newline symbol `shortSummary + "\n" + typeCountSummary` if there are no actions, no?

Reply:
Mh. Okay. Fair enough, I guess.

Personally, I would still add a early-return. I think it makes the code easier to think about if special cases exit the method as soon as possible.